dist/src/main/resources/application.properties

```diff
@@ -27,24 +27,36 @@ management.server.ssl.enabled=false
 spring.web.resources.add-mappings=false
 # General management configuration; disable all endpoints by default but exposes all enabled ones
 # via web. Endpoints should be enabled individually based on the target application
-management.endpoints.enabled-by-default=false
+management.endpoints.access.default=none
 management.endpoints.web.exposure.include=*
 # Health configuration; disable default health indicators. As it's unclear how to do this globally,
 # only the ones which currently would get enabled are explicitly disabled.
-management.endpoint.health.enabled=true
+management.endpoint.health.access=unrestricted
 management.endpoint.health.show-details=always
 management.health.defaults.enabled=false
 # Configure Prometheus as the default enabled backend, for scraping
-management.endpoint.prometheus.enabled=true
+management.endpoint.prometheus.access=unrestricted
 management.prometheus.metrics.export.enabled=true
 # Disable the OpenTelemetry backend by default; can be enabled on demand by users
 management.otlp.metrics.export.enabled=false
 # Allow runtime configuration of log levels
-management.endpoint.loggers.enabled=true
+management.endpoint.loggers.access=unrestricted
 # Allow viewing the config properties, but sanitize their outputs
-management.endpoint.configprops.enabled=true
+management.endpoint.configprops.access=unrestricted
 management.endpoint.configprops.show-values=always
-management.endpoint.info.enabled=true
+management.endpoint.info.access=unrestricted
+# since springboot 3.4, we need to explicitly enable the custom actuator endpoints, as they are globally disabled by default
+management.endpoint.backups.access=unrestricted
+management.endpoint.banning.access=unrestricted
+management.endpoint.clock.access=unrestricted
+management.endpoint.cluster.access=unrestricted
+management.endpoint.exporters.access=unrestricted
+management.endpoint.exporting.access=unrestricted
+management.endpoint.flowControl.access=unrestricted
+management.endpoint.jobstreams.access=unrestricted
+management.endpoint.partitions.access=unrestricted
+management.endpoint.rebalance.access=unrestricted
+management.endpoint.usage-metrics.access=unrestricted
```

Comment:
:disappointed: 

Heavy dislike, but I get that this is the only way to do it :(